Goals.md

```diff
@@ -14,3 +14,4 @@
 - [x] Own a boat
 - [x] Be not famous
 - [ ] Advocate
```

Comment:
yup yup!!

Reply:
yep! looking good what i woudl suggest is 
```suggestion
- [ ] Developer Advocacy here
```